subsys/net/lib/tftp/Kconfig

```diff
@@ -0,0 +1,64 @@
+# Socket MQTT Library for Zephyr
+
+# Copyright (c) 2018 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config TFTP_LIB
+	bool "Socket TFTP Library Support"
+	select NET_SOCKETS
+	select NET_SOCKETS_POSIX_NAMES
+	help
+	  Enable the Zephyr TFTP Library
+
+if TFTP_LIB
+
+module=TFTP
+module-dep=NET_LOG
+module-str=Log level for TFTP
+module-help=Enables tftp debug messages.
+source "subsys/net/Kconfig.template.log_config.net"
+
+config TFTPC_DEFAULT_SERVER_IP
+	string "Default IP address that the TFTP Client will try to connect to."
+	default "192.168.0.100"
+	help
+	  TFTP Client API provides the user a way to specify the IP address of the
+	  TFTP Server. This support is added for cases where the client might want
+	  to communicate with multiple servers. If the user does not specify the IP
+	  address of the TFTP Server, then the library will fallback to this IP.
+
+config TFTPC_DEFAULT_SERVER_PORT
+	int "Default Port that the TFTP Client will try to connect to."
+	default 51000
+	help
+	  This option is used alongside the "TFTPC_DEFAULT_SERVER_IP" to specify the
+	  default port of the TFTP Server.
+
+config TFTPC_MAX_FILENAME_SIZE
+	int "Maximum size of the filename that we can get / put to the TFTP Server."
+	default 10
+	help
+	  Maximum size (defined in the number of characters) of each filename that we
+	  can get or put to the TFTP Server. This is introduced to allow the user to scale up
+	  the size of request array that is not dynamically created (as we don't like dynamic
+	  memory allocations).
+
+config TFTPC_REQUEST_TIMEOUT
+	int "Maximum amount of time the TFTP Client will wait for a response from the server."
+	default 5
+	help
+	  Maximum amount of time (in seconds) that the TFTP Client will wait for data from the
+	  TFTP Server. Once this time has elasped, the TFTP Client will assume that the Server
+	  failed and close the connection.
+
+config TFTPC_REQUEST_RETRANSMITS
+	int "Number of times the TFTP Client will retransmit the request and expect data from the server."
+	default 5
+	help
+	  Once the TFTP Client sends out a request, it will wait TFTPC_REQUEST_TIMEOUT secs for the data
+	  to arrive from the TFTP Server. However, if it doesn't arrive within the given time we will
+	  re-transmit the request to the server in hopes that the server will respond within time to this
+	  request. This number dictates the number of times we will do re-tx of our request before giving up
+	  and exiting.
+
+endif # TFTP_LIB
```

Comment:
you need to break a line after this (it usually breaks make menuconfig without it)

Reply:
Added..